build-tools/create-packs/vs-workload.in.props

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
   <PropertyGroup>
-    <TargetName>Microsoft.NET.Sdk.Android.Workload</TargetName>
+    <TargetName>Microsoft.NET.Sdk.Android.Workload.@VSMAN_VERSION@</TargetName>
```

Comment:
So this would be `Microsoft.NET.Sdk.Android.Workload.7.0`?

https://github.com/xamarin/xamarin-android/blob/e52c131543580641a8884866af2e6aa1e1e02898/Directory.Build.props#L4

That's probably fine, but do we need the `.0`? If this would never be seen by a customer, probably doesn't matter.

Reply:
I like `net7.0` as a suffix.

Reply:
Agreed, it makes the `7.0` less arbitrary.